cmd/productinfo/main.go

```diff
@@ -167,6 +176,8 @@ func infoers() map[string]productinfo.ProductInfoer {
 			infoer, err = azure.NewAzureInfoer(viper.GetString(azureSubscriptionId))
 		case Oracle:
 			infoer, err = oci.NewInfoer()
+		case Alibaba:
+			infoer, err = alibaba.NewAlibabaInfoer(viper.GetString(alibabaRegionId), viper.GetString(alibabaAccessKeyId), viper.GetString(alibabaAccessKeySecret))
```

Comment:
Could we get the credential information from the environment vars?

Reply:
They don't support (yet).

Reply:
Please check out this: https://github.com/spf13/viper#working-with-environment-variables